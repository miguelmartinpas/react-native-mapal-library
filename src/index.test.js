import React from 'react';
import 'react-native';
import { shallow } from 'enzyme';
import { ByeWorld, HelloWorld, ThumbnailCarousel } from './index';

describe('index', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const component = shallow(<ByeWorld />);
      expect(component).toMatchSnapshot();
    });

    it('should match to snapshot', () => {
      const component = shallow(<HelloWorld />);
      expect(component).toMatchSnapshot();
    });

    it('should match to snapshot', () => {
      const component = shallow(<ThumbnailCarousel />);
      expect(component).toMatchSnapshot();
    });
  });
});
