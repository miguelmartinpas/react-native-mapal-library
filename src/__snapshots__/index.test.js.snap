// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index Rendering should match to snapshot 1`] = `ShallowWrapper {}`;

exports[`index Rendering should match to snapshot 2`] = `ShallowWrapper {}`;

exports[`index Rendering should match to snapshot 3`] = `ShallowWrapper {}`;
